gitless/cli/gl_pull.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Gitless - a version control system built on top of Git
+# Licensed under MIT
+
+"""gl pull - A wrapper around git-pull(1)."""
+
+
+from ..import core
+
+
+def parser(subparsers, _):
+    """Adds the pull parser to the given subparsers object."""
+    desc = 'Synchronize branches, tags, references and other meta data from mirror repository.\n' \
+           'For more information on this advanced command refer to the manual page for git-pull.'
```

Comment:
The man page states:
```
       Incorporates changes from a remote repository into the current branch. 
       In its default mode, git pull is shorthand for git fetch followed by git merge FETCH_HEAD.
```

I think we should mention that a `merge` operation will be performed. 
Maybe change "Synchronize" to "Synchronize and merges"?

Reply:
Done.